build.sbt

```diff
@@ -0,0 +1,61 @@
+import uk.gov.hmrc._
+import DefaultBuildSettings._
+import scoverage.ScoverageKeys
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
+import play.sbt.PlayImport.PlayKeys
+import play.sbt.routes.RoutesKeys.{InjectedRoutesGenerator, routesGenerator}
+import sbt.Keys._
+import sbt.Tests.{Group, SubProcess}
+import sbt._
+import scoverage.ScoverageKeys
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin._
+import TestPhases._
+import uk.gov.hmrc.versioning.SbtGitVersioning
+import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
+
+
+val appName: String = "ers-file-validator"
+
+lazy val appDependencies: Seq[ModuleID] = AppDependencies.apply()
+lazy val plugins: Seq[Plugins] = Seq(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin, SbtArtifactory)
+lazy val playSettings: Seq[Setting[_]] = Seq.empty
```

Comment:
I'm not sure if this value is required since it doesn't seem to contain any settings?